### DISCUSSION TITLE
Drone Vision-based Tracking a Mobile Robot in ROS2

### DISCUSSION THREAD

**Discussion by tirpudevk111** [Robotisim/drones_ROS2#10] [Q&A] [answered]
Hi @noshluk2 In the "p1_b_tb3_follower.cpp" linear and angular velocities are published on the topic "/drone/cmd_vel", but you have not mentioned any subscriber to that topic. And how drone understands the data received on "/drone/cmd_vel" this topic? Which firmware did you used px4 or Ardupilot?

**Comment by noshluk2** [comment, marked as answer]
That is why because we are using sjtu drone package that already is subscribes to specific topic
